src/main/java/strings/WorkWithStrings.java

```diff
@@ -0,0 +1,57 @@
+package strings;
+
+public class WorkWithStrings {
+    private int showingLetters;
+
+    public WorkWithStrings() {
+        this.showingLetters = 0;
+    }
+
+    public int findSymbolOccurance(String line, char symbol) {
+        int symbolindaex = -1;
+        symbolindaex = line.indexOf(symbol);
+        return symbolindaex;
+    }
+
+    public int findWordPosition(String source, String target) {
+        int firstIndex = -1;
+        if (source.contains(target))
+            firstIndex = source.indexOf(target.charAt(0));
+        return firstIndex;
+    }
+
+    public String stringReverse(String source) {
+        String reverse = "";
+        for (var i = source.length() - 1; i > -1; i--)
+            reverse += source.charAt(i);
+        return reverse;
+    }
+
+    public boolean isPalindrome(String source) {
+        String reverse = stringReverse(source);
+        if (reverse.toUpperCase().equals(source.toUpperCase()))
```

Comment:
```suggestion
        return reverse.toUpperCase().equals(source.toUpperCase());
```
Можна так, трошки менше коду